.github/workflows/symbolcollision.yml

```diff
@@ -0,0 +1,25 @@
+name: symbolcollision
+
+# Tests the Pods listed in SymbolCollisionTest/Podfile for symbol collisions.
+
+on:
+  pull_request:
+    paths:
+    - '.github/workflows/symbolcollision.yml'
+    - 'SymbolCollisionTest/**'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+  installation-test:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Prereqs
+      run: scripts/install_prereqs.sh FirebasePod iOS
+    - name: Build
+      run: scripts/build.sh FirebasePod iOS
```

Comment:
Does this need to be `SymbolCollision` instead of `FirebasePod`?

Reply:
Yes. Thanks!